backend/tests/integration/submissions/test_submissions_views.py

```diff
@@ -1284,6 +1285,14 @@ def test_conference_raise_loader_error(ticket_mock, inspire_app):
     assert response.status_code == 400
 
 
+REQUIRED_SEMINAR_RECORD_DATA = {
+    "title": {"title": "The Cool Seminar"},
+    "inspire_categories": [{"term": "Accelerators"}],
+    "speakers": [{"name": "Urhan, Ahmet"},],
+    "end_datetime": "2020-05-06T12:30:00.000000",
+    "start_datetime": "2020-05-06T06:30:00.000000",
+    "timezone": "Europe/Zurich",
+}
 SEMINAR_RECORD_DATA = {
```

Comment:
you seem to have the same data for ```REQUIRED_SEMINAR_RECORD_DATA``` for the required fields, you could use them from there and not repeat them

Reply:
well some are same, some different like `field_of_interest`, so I don't wanna make this 
```
{
**REQUIRED_SEMINAR_RECORD_DATA,
override some present fields,
new fields,
}
```

or
```
{
"present_field": REQUIRED_SEMINAR_RECORD_DATA["present_field"]
}
```
for all.

Reply:
for me that way is more clear for test input.